packages/expo-ui/src/jetpack-compose/BottomSheet/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { requireNativeView } from 'expo';
+import { NativeSyntheticEvent } from 'react-native';
+
+export type BottomSheetProps = {
+  /**
+   * The children of the `BottomSheet` component.
+   */
+  children: any;
```

Comment:
could we make this a specific react type?

Reply:
Done!